src/target/page/TargetPage.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import api, { Target } from '../../common/utils/api/api';
+import TargetPage from './TargetPage';
+
+const TEST_TARGET: Target = {
+  id: 'tprek:1000',
+  name: 'Test target',
+  address: 'Main street, Helsinki',
+  description: 'Test target description',
+};
+
+describe(`<TargetPage />`, () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('should show loading indicator', async () => {
+    jest.spyOn(api, 'getTarget').mockImplementation(() =>
+      // eslint-disable-next-line prefer-promise-reject-errors
```

Comment:
Is this eslint-disable-next-line necessary? eslint did not complain when I tried removing it